static/bidder-params/criteo.json

```diff
@@ -1,30 +1,50 @@
-{
-    "$schema": "http://json-schema.org/draft-04/schema#",
-    "title": "Criteo adapter params",
-    "description": "The schema to validate Criteo specific params accepted by Criteo adapter",
-    "type": "object",
-    "properties": {
-        "zoneid": {
-            "type": "number",
-            "description": "Impression's zone ID.",
-            "minimum": 0
-        },
-        "networkid": {
-            "type": "number",
-            "description": "Impression's network ID.",
-            "minimum": 0
-        }
-    },
-    "anyOf": [
-        {
-            "required": [
-                "zoneid"
-            ]
-        },
-        {
-            "required": [
-                "networkid"
-            ]
-        }
-    ]
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Criteo adapter params",
+    "description": "The schema to validate Criteo specific params accepted by Criteo adapter",
+    "type": "object",
+    "properties": {
+        "zoneid": {
+            "type": "integer",
+            "description": "Impression's zone ID.",
+            "minimum": 0
+        },
+        "zoneId": {

```

Comment:
There should only be one zoneId and one networkId parameter, correct - rather than 2 of each?